src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+import java.util.List;
+
+import duke.task.Task;
+
+/**
+ * <code>Ui</code> class handles the interactions with the user.
+ */
+public class Ui {
+
+    /**
+     * Constructor for Ui class.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Generates a welcome message when the Duke application starts up.
+     *
+     * @return Welcome message.
+     */
+    public String welcome() {
+        String logo = "               ____        _        \n"
+                + "               |  _ \\ _   _| | _____\n"
+                + "               | | | | | | | |/ / _ \\\n"
+                + "               | |_| | |_| |   <  __/\n"
+                + "               |____/ \\__,_|_|\\_\\___|\n";
+        return "Hey! I'm\n" + logo + "\n     How may I help you?";
+    }
+
+    /**
+     * Generates a goodbye message when the
+     * user terminates the Duke application.
+     *
+     * @return Goodbye message.
+     */
+    public String exit() {
+        return "Bye. Hope to see you again soon!";
+    }
+
+    /**
+     * Generates the list of existing tasks when user input "list" command.
+     *
+     * @param list List of existing tasks.
+     * @return String message of the list of existing tasks.
+     */
+    public String printTaskList(List<Task> list) {
+        if (list.size() == 0) {
+            return "There is currently no task in your list.";
+        } else {
+            String firstReply = "Here are the tasks in your list:\n";
+            StringBuilder stringBuilder = new StringBuilder();
+
+            for (int i = 0; i < list.size(); i++) {
+                int number = 1 + i;
+                stringBuilder.append("     ").append(number).append(". ").append(list.get(i)).append("\n");
```

Comment:
Perhaps consider using the + operator for better readability? (As of Java 8 and onwards I believe it's converted by the compiler, so you shouldn't lose out on performance.)